server/src/main/kotlin/com/localove/entities/Person.kt

```diff
@@ -0,0 +1,60 @@
+package com.localove.entities
+
+import com.localove.Identifiable
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import javax.persistence.*
+
+@Entity
+@Table(name = "users")
+class Person(
+
+    @Column(name = "login")
+    val login: String,
+
+    @Column(name = "name")
+    val name: String,
+
+    @Column(name = "gender")
+    @Enumerated(EnumType.STRING)
+    val gender: Gender,
+
+    @Column(name = "birth_date")
+    val birthDate: LocalDate,
+
+    @Column(name = "status")
+    val status: String,
+
+    @Column(name = "bio")
+    val bio: String,
+
+    @ManyToOne
+    @JoinColumn(name = "avatar_id")
+    val avatar: Photo
```

Comment:
До первой авторизации аватарки нет
```suggestion
    val avatar: Photo?
```